examples/get-started/pure-js/app.js

```diff
@@ -14,43 +13,24 @@ const INITIAL_VIEW_STATE = {
   pitch: 30
 };
 
-// Set your mapbox token here
-mapboxgl.accessToken = process.env.MapboxAccessToken; // eslint-disable-line
-
-const map = new mapboxgl.Map({
-  container: 'map',
-  style: 'mapbox://styles/mapbox/light-v9',
-  // Note: deck.gl will be in charge of interaction and event handling
-  interactive: false,
-  center: [INITIAL_VIEW_STATE.longitude, INITIAL_VIEW_STATE.latitude],
-  zoom: INITIAL_VIEW_STATE.zoom,
-  bearing: INITIAL_VIEW_STATE.bearing,
-  pitch: INITIAL_VIEW_STATE.pitch
-});
-
 export const deck = new Deck({
-  canvas: 'deck-canvas',
   width: '100%',
   height: '100%',
   initialViewState: INITIAL_VIEW_STATE,
   controller: true,
-  onViewStateChange: ({viewState}) => {
-    map.jumpTo({
-      center: [viewState.longitude, viewState.latitude],
-      zoom: viewState.zoom,
-      bearing: viewState.bearing,
-      pitch: viewState.pitch
-    });
-  },
   layers: [
     new GeoJsonLayer({
-      data: US_MAP_GEOJSON,
+      data: GEOJSON,
       stroked: true,
       filled: true,
       lineWidthMinPixels: 2,
       opacity: 0.4,
-      getLineColor: [255, 100, 100],
-      getFillColor: [200, 160, 0, 180]
+      getLineColor: () => [255, 100, 100],
```

Comment:
Just curious why? You are the biggest proponent of constants...

Reply:
Git is confused due to the renaming of directories. I'll make sure all sample code are aligned.